Remotion/Web/Core/UI/Controls/DatePickerPage.cs

```diff
@@ -52,6 +54,34 @@ namespace Remotion.Web.UI.Controls
 /// </remarks>
 public class DatePickerPage : Page
 {
+#if !NETFRAMEWORK
```

Comment:
Am thinking it would be easier to just make the resource value assignment conditional?